deploy/config-spec/properties.yaml

```diff
@@ -722,7 +722,7 @@ properties:
         type: "string"
       defaultValues:
         - fromVersion: "0.0.0"
-          value: "[{\"path\":\"/stackable/var/druid/segment-cache\",\"maxSize\":\"300g\"}]"
+          value: "[{\"path\":\"/stackable/var/druid/segment-cache\",\"maxSize\":\"900m\"}]"
```

Comment:
Actually i would prefer not to change this, as this reduces the amount you can possible query on a historical.
I have made the observation that only data residing on local disk cache can be queried

Reply:
Reverted.